ChangeLog.md

```diff
@@ -1,4 +1,12 @@
 # Branch Android SDK change log
+- v5.7.2
+  * _*Master Release*_ - Oct 20, 2023
+    - Added additional fields to initialization and event requests
+- v5.7.1
```

Comment:
was this not in master already?

Reply:
Make sure to pull latest from master

Reply:
No because, the release was done off of the release branch instead of master.